File: Client/src/app/shared/models/dashboard.model.ts
export interface IDashboard{
    project: projectAnalist;
}

export class projectAnalist{
    totalProjects: number = 0;
    totalProjectImages: number = 0;
    totalImages: number = 0;
    imagesTagged: number = 0;
    imagesHadQc: number = 0;
    totalTaggedTime: string = '0 hours 0 minutes';
    totalTags: number = 0;
    totalTagsHaveClass: number = 0;
    currentProgress: UserProject = new UserProject();
    userProjects: UserProject[] = [];
}

export class project{
    id:string;
    name: string;
}

export class UserProject{
    userId: number;
    userName: string;
    email: string;
    roleNames: string[];
    taggedTime: string;
    totalTags: number;
    totalQcs: number;
    tagsHaveClass: number;
    imagesHaveTag: number;
    imagesHaveTagClass: number;
}

File: ApiServer/Model/views/DashboardModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ApiServer.Model.views
{
    public static class DashboardModel
    {
        public class Dashboard
        {
            public ProjectAnalist Project { get; set; }
        }

        public class ProjectAnalist
        {
            public int TotalProjects { get; set; }
            public int TotalProjectImages { get; set; }
            public int TotalImages { get; set; }
            public int ImagesTagged { get; set; }
            public int ImagesHadQc { get; set; }
            public string TotalTaggedTime { get; set; }
            public int TotalTags { get; set; }
            public int TotalTagsHaveClass { get; set; }
            public UserProject CurrentProgress { get; set; }
            public List<UserProject> UserProjects { get; set; }
        }

        public class Project
        {
            public Guid Id { get; set; }
            public string Name { get; set; }
        }

        public class UserProject
        {
            public long UserId { get; set; }
            public string UserName { get; set; }
            public string Email { get; set; }
            public string[] RoleNames { get; set; }
            public string TaggedTime { get; set; }
            public int TagsHaveClass { get; set; }
            public int TotalTags { get; set; }
            public int TotalQcs { get; set; }
            public int ImagesHaveTag { get; set; }
        }
        
    }
}
